src/js/media-controller.js

```diff
@@ -496,8 +499,29 @@ const getMediaUIAttributesFrom = (child) => {
   );
 };
 
+const hasMediaUIProps = (mediaStateReceiverCandidate) => {
+  if (
+    mediaStateReceiverCandidate.nodeName?.includes('-')
+    && !!window.customElements.get(mediaStateReceiverCandidate.nodeName?.toLowerCase())
+    && !(mediaStateReceiverCandidate instanceof window.customElements.get(mediaStateReceiverCandidate.nodeName.toLowerCase()))
+  ) {
+    window.customElements.upgrade(mediaStateReceiverCandidate);
```

Comment:
Not sure if this line will survive as it is, but is it safe to assume the custom element here is defined yet? (e.g. the script has been loaded and been processed) This seems to assume so.

Reply:
`upgrade()` will "force" a define for a custom element that's registered (which we identify via the `customElements.get()`, above).